tests/snappi/test_vlan_traffic.py

```diff
@@ -0,0 +1,140 @@
+import pytest
+from pprint import pprint as pprint
+import time
+
+vlan_id = "10"
+macs = ["00:1A:C5:00:00:01", "00:1B:6E:00:00:01"]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def discovery(npu):
+    npu.objects_discovery()
+
+
+def test_l2_untagged_vlan_traffic(npu, dataplane):
+    """
+    Creates vlan 10 and adds two ports to the vlan 10 member
+    and validates the config using l2 traffic
+    """
+    commands = [
+        {
+            "name": "vlan_10",
+            "op": "create",
+            "type": "SAI_OBJECT_TYPE_VLAN",
+            "attributes": ["SAI_VLAN_ATTR_VLAN_ID", "10"],
+        },
+        {
+            "name": "vlan_member_2",
+            "op": "create",
+            "type": "SAI_OBJECT_TYPE_VLAN_MEMBER",
+            "attributes": [
+                "SAI_VLAN_MEMBER_ATTR_VLAN_ID",
+                "$vlan_10",
+                "SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID",
+                "$BRIDGE_PORT_2",
+                "SAI_VLAN_MEMBER_ATTR_VLAN_TAGGING_MODE",
+                "SAI_VLAN_TAGGING_MODE_UNTAGGED",
+            ],
+        },
+        {
+            "name": "PORT_2",
+            "op": "set",
+            "type": "SAI_OBJECT_TYPE_PORT",
+            "attributes": [
+                "SAI_PORT_ATTR_ADMIN_STATE",
+                "true",
+            ],
+        },
+        {
+            "name": "vlan_member_3",
+            "op": "create",
+            "type": "SAI_OBJECT_TYPE_VLAN_MEMBER",
+            "attributes": [
+                "SAI_VLAN_MEMBER_ATTR_VLAN_ID",
+                "$vlan_10",
+                "SAI_VLAN_MEMBER_ATTR_BRIDGE_PORT_ID",
+                "$BRIDGE_PORT_3",
+                "SAI_VLAN_MEMBER_ATTR_VLAN_TAGGING_MODE",
+                "SAI_VLAN_TAGGING_MODE_UNTAGGED",
+            ],
+        },
+        {
+            "name": "PORT_3",
+            "op": "set",
+            "type": "SAI_OBJECT_TYPE_PORT",
+            "attributes": [
+                "SAI_PORT_ATTR_ADMIN_STATE",
+                "true",
+            ],
+        },
+    ]
+    for command in commands:
+        print("-" * 80)
+        print(command)
+        result = npu.command_processor.process_command(command)
+        print(result)
+    try:
+        macs = ["00:11:11:11:11:11", "00:22:22:22:22:22"]
```

Comment:
Please put traffic related part under `if npu.run_traffic:` condition. Please see https://github.com/opencomputeproject/SAI-Challenger/blob/89eb3401b12ad33d0499706df8f3868ffd47c07b/tests/test_l2_basic.py#L47

Reply:
implemented this fix